bloom.go

```diff
@@ -135,6 +135,11 @@ func (f *BloomFilter) K() uint {
 	return f.k
 }
 
+// BitSet returns the underlying bit set for this filter.
```

Comment:
I think that we often call the data structure in question a bitset (see https://www.cplusplus.com/reference/bitset/bitset/) and not a 'bit set'.

Would you consider tweaking this comment?

Reply:
Oh, I see you mean `bit set` - yes I agree and you fixed it already :)